tests/Platforms/SqlitePlatformTest.php

```diff
@@ -17,7 +17,10 @@ class SqlitePlatformTest extends AbstractPlatformTestCase
 {
     public function createPlatform(): AbstractPlatform
     {
-        return new SqlitePlatform();
+        $platform =  new SqlitePlatform();
+        $platform->disableSchemaEmulation();
+
+        return $platform;
```

Comment:
Is this change still relevant?

Reply:
Oh no, you're right, let me restore the file to its original state.